orga/app/components/team/members-list.js

```diff
@@ -3,4 +3,8 @@ import Component from '@glimmer/component';
 
 export default class MembersList extends Component {
```

Comment:
le nom du fichier ne contenait pas de S , il n'était pas lié a son `.hbs`. le this.currentUser.isAdminInorganization n'était donc jamais trouvé dans le `.hbs`